godtools/App/Share/Data/TranslationsRepository/ManifestParser/ParseTranslationManifestForRenderer.swift

```diff
@@ -19,10 +19,16 @@ class ParseTranslationManifestForRenderer: TranslationManifestParser {
         ParserConfigKt.FEATURE_MULTISELECT
     ]
         
-    init(resourcesFileCache: ResourcesSHA256FileCache) {
-    
+    init(appConfig: AppConfig, resourcesFileCache: ResourcesSHA256FileCache) {
+            
+        let parserConfig = ParserConfig()
+            .withParsePages(enabled: true)
+            .withParseTips(enabled: true)
```

Comment:
these default to true, so you technically don't need to specify them unless you want to be verbose.

Reply:
That's good to know, I think I will keep them just to be verbose.